packages/app/src/cli/commands/app/versions/list.ts

```diff
@@ -27,6 +27,13 @@ export default class VersionsList extends Command {
       env: 'SHOPIFY_FLAG_CLIENT_ID',
       exclusive: ['config'],
     }),
+    reset: Flags.boolean({
```

Comment:
Can you elaborate on why this is necessary? Why not always treat as `reset: false`?